jenkins/config.sls

```diff
@@ -0,0 +1,28 @@
+{% from "jenkins/map.jinja" import jenkins with context %}
+include:
+  - jenkins
+
+get_jenkins_config_from_git:
+  git.latest:
+    - name: {{ jenkins.jenkins_config_git_repo }}
+    - target: {{ jenkins.home }}
+    - force_clone: True
+    - require:
+      - sls: jenkins
+
+change_file_ownership_of_JENKINS_HOME:
+  file.directory:
+    - name: {{ jenkins.home }}
+    - user: {{ jenkins.user }}
+    - group: {{ jenkins.group }}
+    - recurse:
+      - user
+      - group
+    - require:
+      - git: get_jenkins_config_from_git
+
+restart_jenkins_with_systemctl:
+  cmd.run:
```

Comment:
When I added the require_in or require statements, it somehow completely messes up the order of execution. So I did this because it works. But I know that this is not the best solution and should be avoided since it does not follow the DRY principle. Do you suggest I should use the same service.running statement in the init.sls file? How can I implement this without messing around with the correct execution order?